app/components/task-selection/component.js

```diff
@@ -112,6 +121,18 @@ export default Component.extend({
       return task && task.get('project.customer') || this.get('_customer')
     },
     set(value) {
+      /**
+       * It is also possible a activity/report was selected from the history.
+       * An activity/report has a duration property, check for this one.
+       */
+      if (value !== null && value.get('duration')) {
+        this.set('task', value.get('task'))
+        this.set('_customer', value.get('task.project.customer'))
+        this.set('_project', value.get('task.project'))
+
+        return this.get('_customer')
```

Comment:
According to the Ember computed documentation, the computed setter should return the new value. As this is the customer property, we return the customer.